owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt

```diff
@@ -224,6 +224,9 @@ abstract class DrawerActivity : ToolbarActivity() {
         // Allow or disallow touches with other visible windows
         getBottomNavigationView()?.filterTouchesWhenObscured = PreferenceUtils.shouldDisallowTouchesWithOtherVisibleWindows(this)
         if (account != null) {
+            capabilitiesViewModel.capabilities.value?.let {
+                setSpacesVisibilityBottomBar(it.peekContent())
+            }
             capabilitiesViewModel.capabilities.observe(this, Event.EventObserver { uiResult: UIResult<OCCapability> ->
                 setSpacesVisibilityBottomBar(uiResult)
             })
```

Comment:
Try this to handle the capabilities value as many times as needed. EventObserver works as OneTime value observer.
```suggestion
            capabilitiesViewModel.capabilities.observe(this) { event: Event<UIResult<OCCapability>> ->
                setSpacesVisibilityBottomBar(event.peekContent())
            }

Reply:
True, nice one!